rtos/FreeRTOSConfig.h

```diff
@@ -82,7 +83,7 @@
 /* Memory allocation related definitions. */
 #define configSUPPORT_STATIC_ALLOCATION         0
 #define configSUPPORT_DYNAMIC_ALLOCATION        1
-#define configTOTAL_HEAP_SIZE                   (230*1024) // figure out what linker will accept
+#define configTOTAL_HEAP_SIZE                   (192*1024) // figure out what linker will accept
```

Comment:
Was this bombing out with the larger RTOS heap?

Reply:
Yeah, .bss region overflows RAM. This was the highest I could set it and make it work.

Reply:
Okay - this highlights a future todo - there are some HW specific things in `FreeRTOSConfig.h` that we should figure out how to abstract, maybe another header file in the hardware folder with some of these variables defined. Seems like a tomorrow problem.